src/app/routes/cpsAsset/getInitialData/processUnavailableMedia/index.js

```diff
@@ -2,21 +2,44 @@ import pathOr from 'ramda/src/pathOr';
 import assocPath from 'ramda/src/assocPath';
 import path from 'ramda/src/path';
 import nodeLogger from '#lib/logger.node';
-import { NO_MEDIA_BLOCK } from '#lib/logger.const';
+import {
+  NO_MEDIA_BLOCK,
+  MEDIA_ASSET_EXPIRED,
+  MEDIA_ASSET_REVOKED,
+  MEDIA_METADATA_UNAVAILABLE,
+} from '#lib/logger.const';
 
 const logger = nodeLogger(__filename);
 
-const UNAVAILABLE_MEDIA_TEXT = 'unavailableMedia';
-const REVOKED_MEDIA = 'external_vpid';
+export const UNAVAILABLE_MEDIA_TEXT = 'unavailableMedia';
+export const EXTERNAL_VPID = 'external_vpid';
 export const unavailableMediaBlock = {
   type: UNAVAILABLE_MEDIA_TEXT,
   model: {},
   id: UNAVAILABLE_MEDIA_TEXT,
 };
 
+export const logUnavailableMedia = blocks => {
+  const mediaBlock = blocks.find(block => block.type === EXTERNAL_VPID);
+  if (mediaBlock) {
+    const statusCode = path(['statusCode'], mediaBlock);
+    switch (statusCode) {
+      case 404:
+        logger.warn(MEDIA_ASSET_REVOKED);
```

Comment:
does `logger.warn` add more info to this, eg, which page it was?

Worried this might not be helpful - the other calls to `logger` in this file include the page URL where the error happened: https://github.com/bbc/simorgh/pull/6294/files#diff-3b008fba57fe9eb2546a7ae54d193807R53